qiita_db/metadata_template.py

```diff
@@ -1232,6 +1287,93 @@ def study_id(self):
         """
         return self._id
 
+    def remove_category(self, category):
+        """Remove a category from the sample template
+
+        Parameters
+        ----------
+        category : str
+            The category to remove
+
+        Raises
+        ------
+        QiitaDBColumnError
+            If the column does not exist in the table
+        """
+        table_name = self._table_name(self.study_id)
+        conn_handler = SQLConnectionHandler()
+
+        if category not in self.categories():
+            raise QiitaDBColumnError("Column %s does not exist in %s" %
+                                     (category, table_name))
+
+        # This operation may invalidate another user's perspective on the
```

Comment:
If user A is looking at study FOO, and user B deletes a column out of FOO, the user A's view is incorrect.